app/repositories/application/GeneralApplicationRepository.scala

```diff
@@ -24,16 +24,17 @@ import model.ApplicationStatuses._
 import model.AssessmentScheduleCommands.{ ApplicationForAssessmentAllocation, ApplicationForAssessmentAllocationResult }
 import model.Commands._
 import model.EvaluationResults._
-import model.Exceptions.{ ApplicationNotFound, CannotUpdatePreview }
+import model.Exceptions.{ApplicationNotFound, CannotUpdatePreview}
+import model.OnlineTestCommands.{OnlineTestApplication, TimeAdjustmentsOnlineTestApplication}
 import model.PersistedObjects.ApplicationForNotification
 import model._
 import model.command._
 import org.joda.time.format.DateTimeFormat
-import org.joda.time.{ DateTime, LocalDate }
+import org.joda.time.{DateTime, LocalDate}
 import play.api.Logger
-import play.api.libs.json.{ Format, JsNumber, JsObject }
-import reactivemongo.api.{ DB, QueryOpts, ReadPreference }
-import reactivemongo.bson.{ BSONDocument, _ }
```

Comment:
I believe we decided on spaces before and after curly braces in imports, it's not a default intellij option (but it is the scalariform default) - you'll need to enable this option if i've remembered it the right way round:

![selection_133](https://cloud.githubusercontent.com/assets/82004/18547995/45147fee-7b3e-11e6-8271-a27df7137e7c.png)